Tests/FunctionalTests/fss-search-page.spec.ts

```diff
@@ -41,14 +41,16 @@ describe('Test Search Attribute Scenario On Search Page', () => {
   })
   
   it('Verify if click search button without selecting a field value', async () => {
+    await page.reload();
```

Comment:
These `page.reload()`s should be removed and the setup and teardowns split so only the browser is launched/closed in the "all" methods

Reply:
Removed from the scripts. Thanks.